### PR TITLE
added profile page for all authors

### PR DESCRIPTION
**Pull request by AidanHoremans**

### CODE REVIEW

File: frontend/src/api/author_api.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
```
Comment (laingkob) [inline frontend/src/api/author_api.js:1]: I already wrote this in post_display_api.js so I can remove it from there and we can have them in separate files, probably more visible

File: frontend/src/components/ListItems/list-items.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```
Comment (laingkob) [inline frontend/src/components/ListItems/list-items.test.js:1]: This doesn't actually test list-items, so we should note that somewhere or let this test fail for now

File: frontend/src/components/Posts/post-plain.js [hunk 3]
```diff
@@ -3,32 +3,38 @@ import { useState } from "react";
...
-            </div>
-            {/* Will need to handle other post types here, plain for now */}
-            <div className="content-container">
-                <h5>{data["post"]["title"]}</h5>
-                <div className="content">
-                    {data["post"]["content"]}
+        <div>
+            <div className="message">
+                <div className="from">
+                    <h6><a href={authorUrl}>{data.post.author.displayName}</a></h6>
```
Comment (laingkob) [inline frontend/src/components/Posts/post-plain.js:18]: Would it be intuitive enough to use the image as a link to the author profile?

File: frontend/src/components/ListItems/list-items.test.js (hunk 2 above)
Comment (AidanHoremans) [reply frontend/src/components/ListItems/list-items.test.js:1]: None of my tests do anything for now. I feel like letting them fail is enough of a reminder.

File: frontend/src/components/Posts/post-plain.js (hunk 3 above)
Comment (AidanHoremans) [reply frontend/src/components/Posts/post-plain.js:18]: I personally dont think it matters right now, but thats something we can likely add.

File: frontend/src/pages/Profile/profile.test.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
```
Comment (laingkob) [inline frontend/src/pages/Profile/profile.test.js:1]: Again I think we should note that this test isn't accurate or fail it for now

**Review by laingkob — APPROVED** [review]

**Review by laingkob — APPROVED** [review]

File: frontend/src/components/Posts/post-plain.js (hunk 3 above)
Comment (laingkob) [reply frontend/src/components/Posts/post-plain.js:18]: Yeah, true, that's more of a detail we can worry about later

File: frontend/src/components/ListItems/list-items.test.js (hunk 2 above)
Comment (laingkob) [reply frontend/src/components/ListItems/list-items.test.js:1]: Okay, sounds good.

**Comment by laingkob** [comment]
Can't even tell it's your first time using React, great work using its tools

**Pull request merged**